src/serialize-spec.ts

```diff
@@ -123,16 +127,7 @@ function serializeNode(node: Node): any {
       };
     }
     case 'root': {
-      return {
-        type: 'root',
-        input: Object.fromEntries(
-          // sort by keys so we always get the same serialization for the same spec
-          // will be important for hashing
-          Object.entries(node.input)
-            .sort((a, b) => a[0].localeCompare(b[0]))
-            .map(([key, input]) => [key, serializeInput(input)])
-        ),
-      };
+      return { type: 'root' };
```

Comment:
changed root serialization to simply be `{ type: 'root' }`, and we can still deserialize by having an additional parameter for the inputs